### ISSUE TITLE
Debian repo not found?

### ISSUE THREAD

**Issue by joshbowyer** [ProtonVPN/linux-app#15]
I did an apt update today and it said the ProtonVPN repo is not found, so then I went to repo.protonvpn.com/debian and it is 404'd?

**Comment by joshbowyer** [comment]
Ok, adding the / in the URL works in the browser, but the error from apt says it is looking for 'main/binary-arm64' which doesnt exist in the repo... shouldnt it be looking for binary-all?

**Comment by calexandru2018** [comment]
Hey @joshbowyer 

That is actually your systems that is configured to search for arm64, which makes me believe we're running on a arm device.

**Comment by joshbowyer** [comment]
Yes that is correct. This wasnt a problem a few days ago.

**Comment by calexandru2018** [comment]
@joshbowyer in terms of repo nothing has changed. Maybe some update to the distro that changed it?

**Comment by joshbowyer** [comment]
Nope, I havent even done an apt upgrade since it was working last. The workaround is to add arch=all to the repo file like this:

deb [arch=all, signed-by=blah blah]

https://stackoverflow.com/a/15273416 explains it better, not sure how it wasnt complaining before as nothing changed on either end?
